lib/remote.provider.jstp.api.js

```diff
@@ -79,13 +82,19 @@ const createRemoteProviderJstpApi = (gsProvider, cursorFactory) => ({
         callback(errorCodes.INSUFFICIENT_PERMISSIONS);
         return;
       }
-      gsProvider.get(
-        id,
-        (err, record) => {
-          callback(err && err.code, record);
-        },
-        createPermissionChecker(gsProvider, 'read', userId)
-      );
+      gsProvider
+        .enableLogging({
+          SystemUser: userId,
+          IP: connection.remoteAddress,
+          ProcessToken: common.generateGUID(),
+        })
```

Comment:
Perhaps extract this into a helper function to avoid typing every field key every time?
```js
const enableProviderLogging = (provider, userId, conn) =>
  provider.enableLogging({
    SystemUser: userId,
    IP: conn.remoteAddress,
    ProcessToken: common.generateGUID(),
  });
```